common/src/main/java/net/william278/huskclaims/command/HuskClaimsCommand.java

```diff
@@ -177,8 +178,25 @@ public List<String> suggest(@NotNull CommandUser user, @NotNull String[] args) {
             case 0, 1 -> SUB_COMMANDS.keySet().stream().filter(n -> user.hasPermission(getPermission(n))).toList();
             default -> switch (args[0].toLowerCase(Locale.ENGLISH)) {
                 case "import" -> switch (args.length - 2) {
-                    case 0 -> plugin.getImporters().stream().map(Importer::getName).toList();
-                    case 1 -> List.of("start", "set", "reset");
+                    case 0 -> List.of("database").stream()
+                              .collect(Collectors.toCollection(() -> plugin.getImporters().stream()
+                                      .map(Importer::getName).collect(Collectors.toList())));
```

Comment:
If we're adding DatabaseImporter to the hooks list then we shoudlnt need to manually stream in "database" here as it'll pick it up from the list :)